pykechain/models/widgets/widgets_manager.py

```diff
@@ -540,10 +563,12 @@ def add_filteredgrid_widget(
             }
         )
 
+        meta, title = _set_title(meta, title=title, **kwargs)
+
         if parent_instance_id:
```

Comment:
parent_instance_id is always declared, hence the if statement can be removed.

Reply:
That is not true, if you send an empty `parent_instance_id`, a `ValidationError` from the BE is returned. If `parent_instance_id` is in the keys of the meta, then it needs to have an `UUID` value (that means empty string is also unacceptable).

Reply:
Misunderstanding, parent instance id should be removed from meta if not provided